Acrolinx Batch 7 Edits (#3781)
MicrosoftDocs/azure-docs-cli: docs-ref-conceptual/cli-versioning-identifiers.md

# Differences between Azure CLI products

As of the end of June 2018, explicit version numbers have been removed from Azure CLI product names. This change helps eliminate

> confusion that sometimes showed up in documentation. The users were told to use "the Azure CLI," but it was unclear what version

Reviewer: ```suggestion
confusion when documentation instructed to use "the Azure CLI," but was unclear what version
```

of the product was being referenced. If you're familiar with the old product names, here's how they have changed:

* Azure CLI versions 2.0 and later are now referred to only as "Azure CLI."
* Earlier Azure CLI versions (1.x and lower) are now referred to as "Azure classic CLI."

The name change to Azure classic CLI makes it clear that this tool is meant to be used only with the classic deployment
model. The classic CLI is also no longer updated or maintained. For this reason, and many more, it's recommended that you move any classic

> deployments to use the Azure Resource Manager model. And migrate to the latest available version of the Azure CLI.

Reviewer: ```suggestion
deployments to use the Azure Resource Manager model. Migrate to the latest available version of the Azure CLI.
```
